tests/test_checker.py

```diff
@@ -84,14 +83,17 @@ def test_check(self):
         file_ext_data = self.checker.get_external_data(ExternalData.Type.FILE)
         self.assertEqual(len(file_ext_data), NUM_FILE_IN_MANIFEST)
 
-        archive_ext_data = self.checker.get_external_data(ExternalData.Type.ARCHIVE)
+        archive_ext_data = self.checker.get_external_data(
+            ExternalData.Type.ARCHIVE)
```

Comment:
I don't like that this is to the left of the `=` sign. Does it fit if you bring the `self.checker...` to the line below?